cluster/manifests/skipper/hpa-redis.yaml

```diff
@@ -0,0 +1,56 @@
+{{ if eq .ConfigItems.skipper_ingress_redis_autoscaling "true" }}
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
+metadata:
+  name: skipper-ingress-redis
+  namespace: kube-system
+  labels:
+    application: skipper-ingress-redis
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: StatefulSet
+    name: skipper-ingress-redis
+  minReplicas: {{ .ConfigItems.skipper_ingress_redis_min_replicas }}
+  maxReplicas: {{ .ConfigItems.skipper_ingress_redis_max_replicas }}
+  metrics:
+  - type: Resource
+    resource:
+      name: cpu
+      target:
+        type: Utilization
+        averageUtilization: {{ .ConfigItems.skipper_ingress_redis_target_average_utilization_cpu }}
+  - type: Resource
+    resource:
+      name: memory
+      target:
+        type: Utilization
+        averageUtilization: {{ .ConfigItems.skipper_ingress_redis_target_average_utilization_memory }}
+{{ if ne .ConfigItems.skipper_ingress_redis_cluster_scaling_schedules "" }}
+  {{ range split .ConfigItems.skipper_ingress_redis_cluster_scaling_schedules "," }}
+  {{ $name_target := split . "=" }}
+  - type: Object
+    object:
+      describedObject:
+        apiVersion: zalando.org/v1
+        kind: ClusterScalingSchedule
+        name: {{ index $name_target 0 }}
+      metric:
+        name: {{ index $name_target 0 }}
+      target:
+        averageValue: {{ index $name_target 1 }}
+        type: AverageValue
+  {{ end }}
+{{ end }}
+  behavior:
+    scaleDown:
+      stabilizationWindowSeconds: 600
```

Comment:
What's the motivation for delayed scale-down here? Seems reasonable to just let it scale down as fast as HPA would normally do it.

Reply:
It's copied from skipper-ingress, but honestly I never saw that it stopped downscaling.
In general for regular load spikes it's better to downscale slower.
I would let it like this.

Reply:
For skipper-ingress the motivation was to slow down target group detaching.

here the theory is regular load spikes, but as we don't now the impact without testing we might as well start less conservative and adjust if it's needed (we already have the conservative and low HPA CPU target so we don't need this in both directions up front IMO)